lib/transloadit/request.rb

```diff
@@ -52,15 +52,22 @@ def get(params = {})
   end
 
   #
-  # Performs an HTTP DELETE to the request's URL. Takes an optional hash of
-  # query params.
+  # Performs an HTTP DELETE to the request's URL. Takes an optional hash
+  # containing the form-encoded payload.
   #
-  # @param  [Hash] params additional query parameters
+  # @param  [Hash] payload the payload to form-encode along with the POST
   # @return [Transloadit::Response] the response
   #
-  def delete(params = {})
+  def delete(payload = {})
     self.request! do
-      self.api[url.path + self.to_query(params)].delete(API_HEADERS)
+      # RestClient::Resource does not provide post payload
+      # feature with http delete method
+      RestClient::Request.execute(
+        :method => 'delete',
+        :url => self.api.concat_urls(self.api.url, self.url.path),
+        :payload => self.to_payload(payload),
+        :headers => API_HEADERS
+      )
```

Comment:
Now, that we don't support Ruby 1.9 anymore, we might as well adapt the new hash style:

e.g.:

```ruby
{
  method: 'delete'
}
```

This is obviously outside of scope for this PR, just an additional note :)